src/Avalonia.Base/Media/GlyphRun.cs

```diff
@@ -166,7 +166,7 @@ public GlyphRunMetrics Metrics
         /// </summary>
         public Point BaselineOrigin
         {
-            get => _baselineOrigin ?? default;
+            get => PlatformImpl.Item.BaselineOrigin;
             set => Set(ref _baselineOrigin, value);
```

Comment:
This setter is now useless?

Reply:
No because the Platform impl is rest on Set